.github/workflows/integration-test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_ARN }}
           role-duration-seconds: 1200
-          aws-region: us-west-2
```

Comment:
This will change the region for all subsequent steps. Instead I suggest you change the `AWS_REGION` env var in the login step.

Reply:
Although I'm not sure if the token generated is per region. This is more of a STS question and I'm not sure of the answer. Does the validator need to be ran in us-west-2?

Reply:
I don't think it needs to be ran in us-west-2.  I was changed the token to us-east-1 and the workflow still passed.  However, I think most integ tests run in us-west-2 so I think this change is fine for now.  I tested using the env var for `AWS_REGION` in my fork and it worked as well.